stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -37,10 +38,17 @@
     // local variables
     long iteration = 0;
 
-    public AbstractWorkerTask(OperationSelectorBuilder<O> operationSelectorBuilder) {
+    public AbstractWorkerTask(@Nonnull OperationSelectorBuilder<O> operationSelectorBuilder) {
```

Comment:
Why did you add the @Nonnull thing? Imho terrible syntactical noise.

In nice you could do:

void bla(SomeObject o)

or

void bla(?SomeObject o)

Without ? it means that it needs to be not null and with ? it means it is nullable. 

Anyhow.. before doing something like this.. we should agree as team to use it through the project or not at all.